x-pack/plugins/siem/public/components/timeline/body/renderers/zeek/zeek_signature.tsx

```diff
@@ -95,33 +95,35 @@ export const DraggableZeekElement = pure<{
   ) : null
 );
 
-export const Link = pure<{ value: string | null; link?: string | null }>(({ value, link }) => {
-  if (value != null) {
-    if (link != null) {
-      return (
-        <LinkFlexItem grow={false}>
-          <div>
-            <GoogleLink link={link}>{value}</GoogleLink>
-            <ExternalLinkIcon />
-          </div>
-        </LinkFlexItem>
-      );
+export const Link = pure<{ value: string | null | undefined; link?: string | null }>(
+  ({ value, link }) => {
+    if (value != null) {
+      if (link != null) {
+        return (
+          <LinkFlexItem grow={false}>
+            <div>
+              <GoogleLink link={link}>{value}</GoogleLink>
+              <ExternalLinkIcon />
+            </div>
+          </LinkFlexItem>
+        );
+      } else {
+        return (
+          <LinkFlexItem grow={false}>
+            <div>
+              <GoogleLink link={value} />
+              <ExternalLinkIcon />
+            </div>
+          </LinkFlexItem>
+        );
+      }
     } else {
-      return (
-        <LinkFlexItem grow={false}>
-          <div>
-            <GoogleLink link={value} />
-            <ExternalLinkIcon />
-          </div>
-        </LinkFlexItem>
-      );
+      return null;
```

Comment:
I am wondering if we should just start to return instead of wrapping it under an else

Reply:
Eh...I think it looks ok.